core/src/test/scala/com/pingcap/tispark/datasource/CheckUnsupportedSuite.scala

```diff
@@ -19,56 +19,6 @@ class CheckUnsupportedSuite extends BaseDataSourceTest("test_datasource_check_un
   override def beforeAll(): Unit =
     super.beforeAll()
 
-  test("Test write to table with secondary index: UNIQUE") {
-    dropTable()
-
-    jdbcUpdate(
-      s"create table $dbtable(i int, s varchar(128), UNIQUE (i))"
-    )
-    jdbcUpdate(
-      s"insert into $dbtable values(null, 'Hello')"
-    )
-
-    {
-      val caught = intercept[TiBatchWriteException] {
-        tidbWrite(List(row2, row3), schema)
-      }
-      assert(
-        caught.getMessage
-          .equals(
-            "tispark currently does not support write data table with secondary index(KEY & UNIQUE KEY)!"
-          )
-      )
-    }
-
-    testTiDBSelect(Seq(row1))
-  }
-
-  test("Test write to table with secondary index: KEY") {
```

Comment:
please add a positive test

Reply:
It is covered by AddingIndexSuite.

Reply:
i did not find such test in AddingIndexSuite:
`table with non unique key`

Reply:
Added。